index.js

```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line no-unused-vars
+import formInput from './modules/formInput.js';
```

Comment:
- [ ] Please, consider that this importing is none - there is no such an exporting in the `formInput.js` module. In this case, you can remove this line - it is doing nothing 💯